core/handlers/v2/hoverfly_postserveactiondetails_handler.go

```diff
@@ -0,0 +1,72 @@
+package v2
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/SpectoLabs/hoverfly/core/handlers"
+	"github.com/codegangsta/negroni"
+	"github.com/go-zoo/bone"
+)
+
+type HoverflyPostServeActionDetails interface {
+	GetPostServeActionDetails() PostServeActionDetailsView
+	RegisterPostServeActionHook(string, string, string, int) error
```

Comment:
Perhaps just call it `PostServeAction` instead of `PostServeActionHook`, to me the former is clear enough